requirements.txt

```diff
@@ -6,10 +6,11 @@ chardet==5.1.0
 charset-normalizer==3.1.0
 click==8.1.3
 craft-archives==1.1.3
-craft-cli==2.0.0
+craft-cli==2.1.0
 craft-grammar==1.1.1
-craft-parts==1.21.1
-craft-providers==1.13.0
+#craft-parts==1.21.1
+git+https://github.com/canonical/craft-parts.git@6244b2ba14af7376400e016ae709abc6ebee6fc7#egg=craft_parts
+craft-providers==1.15.0
```

Comment:
FYI 1.15.0 raises the compatibility tag version from v1 to v2.  

I have an upcoming PR to use the configure and remove hooks from charmcraft that clean up old images and instances, but it can be merged later.